lua/toggleterm/terminal.lua

```diff
@@ -110,7 +110,7 @@ local function setup_buffer_autocommands(term)
     table.insert(commands, {
       "CursorHold",
       fmt("<buffer=%d>", term.bufnr),
-      "lua require'toggleterm.ui'.save_window_size()",
+      fmt("lua require'toggleterm.ui'.save_window_size(0, '%s')", term.direction),
```

Comment:
nit: maybe in this case it's better to pass nil to the save window size's first arg and then in the function if the window is nil then use `nvim_get_current_win` so that it uses the window id rather than depending on the `0` is current behaviour

Reply:
I'll switch the order of the two parameters then.